README.md

```diff
@@ -41,9 +41,9 @@ We are willing to answer questions.
 ## Getting Started
 
 ```javascript
-import vjsn from "zome-vjsn";
+import vueSpatialNavigation from "zome-vjsn";
 
-app.use(vjsn, globalConfig);
+app.use(vueSpatialNavigation);
```

Comment:
globalConfig is useful and shouldnt be left out.

Reply:
Yes but not defined in the current code sample